### ISSUE TITLE
How to set minPixelDragDistance

### ISSUE THREAD

**Issue by alanOracle** [JamesLMilner/terra-draw#167]
Hi again @JamesLMilner hope you had a nice holidays and happy new year.

I recently moved the controller I had for terra draw from js to ts. And one of the properties I could not set according to the ts definitions is  `minPixelDragDistance` which i know its a property on the base adapter.

## Base Adapater

https://github.com/JamesLMilner/terra-draw/blob/6b62cecfa04aad460a5c3c1e4fd9950d365e9505/src/adapters/common/base.adapter.ts#L20-L25

## Maplibre Adapater
https://github.com/JamesLMilner/terra-draw/blob/6b62cecfa04aad460a5c3c1e4fd9950d365e9505/src/adapters/maplibre-gl.adapter.ts#L14

## Desired code
the way I'm doing it right now is as follows:

`_terraDrawAdapter = new TerraDrawMapLibreGLAdapter({map, minPixelDragDistance: MIN_PIXEL_DRAG_DISTANCE});`

Im also changing my local `maplibre-gl.adapter.d.ts` to be something like this (which is not the correct way since those files are being gotten by NPM and are ignored on GIT) 
```
constructor(config: {
        map: Map;
        coordinatePrecision?: number;
        minPixelDragDistance?: number;
    });
```

So is there a way of setting that property without changing that file Im not aware of?, and if not I think there should be one.

Thanks. ❤️

**Comment by JamesLMilner** [comment]
Hey @alanOracle - this is a bug in the sense that it should be configurable. Let me fix this for you 👍🏻

**Comment by JamesLMilner** [comment]
This is now resolved in `0.0.1-alpha.53`
